### PR TITLE
Bugfix/release 1.28+ck2

### PR DESCRIPTION
**Pull request by addyess**
In order to prepare for the 1.28+ck2 release, update the release notes with descriptions of the bugs fixed. 

Also updated the generator code to correctly generate release pages

### CODE REVIEW

File: generator/k8s_docs_tools/templates/component.j2 [hunk 1]
```diff
@@ -52,7 +52,7 @@ These charms are the core components of the Charmed Kubernetes release.
 ## CNI charms
 
 These are the Container Network Interface (CNI) charms supported by Charmed Kubernetes
-(Calico is the default CNI and is also included in the core charms table above).
+(calico is the default CNI and is also included in the core charms table above).
```
Comment (evilnick) [inline generator/k8s_docs_tools/templates/component.j2:55]: why are we lowercasing this?

File: generator/k8s_docs_tools/templates/supported-versions.j2 [hunk 2]
```diff
@@ -19,12 +19,21 @@ source of the information - please file a bug at
...
-Supported releases: **{{ supported_releases_x }}**
+Supported releases (click buttons for more information): 
+
+<div class="row">
+{%- for rel in supported_releases %}
+ <div class="col-2">
+  <span class="u-vertically-center"><a href="/kubernetes/docs/{{rel}}/components" class="p-button--positive">{{rel}}.x</a></span>
+ </div>
+{%- endfor %}
+</div>
```
Comment (evilnick) [inline generator/k8s_docs_tools/templates/supported-versions.j2:36]: thanks! I would have got around to this...
I think the buttons are far better at drawing attention to the actual versions.

File: pages/k8s/release-notes.md [hunk 3]
```diff
@@ -14,6 +14,37 @@ toc: False
 ---
 
 <!-- AUTOGENERATE RELEASE NOTES HERE -->
+## 1.28+ck2 Bugfix release
```
Comment (evilnick) [inline pages/k8s/release-notes.md:17]: can we add a blank line here please

**Review by evilnick — APPROVED** [review]
some tweaks needed :lollipop:

File: generator/k8s_docs_tools/templates/component.j2 (hunk 1 above)
Comment (addyess) [reply generator/k8s_docs_tools/templates/component.j2:55]: I lower-cased it to match what was in the current component.md page.  Most of the template changes were made to match what was already there.

**Pull request merged**